code/workspaces/web-app/src/components/assetRepo/editRepoMetadataValidator.js

```diff
@@ -0,0 +1,20 @@
+import validate from 'validate.js';
+
+export const editConstraints = {
+  owners: {
+    presence: { allowEmpty: false },
```

Comment:
This is handy, think we should consider adding it to the DataStore to avoid situations where storage can be un-owned by anyone.

Reply:
Good idea, done